src/affordance_entity.js

```diff
@@ -0,0 +1,117 @@
+//this file will contain the entity class which keeps all info about the affordance and some actions that manipulate the affordance
+import { addMiaIcon, addLoader, deleteLoader, addInfoIcon, addFailed } from "./node_modifications.js";
+import { createEmptyStore, storeSize , getLinkedDataNQuads, addDataToStore} from "./linked_data_store.js";
+import { getBoundryInfo, getInfoPopup } from "./info_extraction.js";
+import Popup from "./popup.js";
+
+export default class AffordanceEntity {
+    constructor(affordance) {
+        this.uri = affordance.href;
+        this.id = 'id-' + Math.random().toString(36).slice(2, 11); // todo: check if this is really needed in the end
+        this.node = affordance;
+        this.store = createEmptyStore();
```

Comment:
what is this store about? every variable is there to 'store' something -- better to consider a name that hints at what it is about.